src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -372,16 +410,16 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             }
         }
 
-        // The same goes for consolidated actions.
-        if (consolidatedCmd)
+        // Update maskedCmd:
+        if (maskedCmd)

```

Comment:
Comment suggestion for **why** we want to do this (because we just updated oldCmd, why would we have an old and a masked? explain!)